config/rbac/role.yaml

```diff
@@ -6,7 +6,9 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
-- resources:
+- apiGroups:
+  - ""
```

Comment:
Why is this needed ?

Reply:
This fixes things for Openshift clusters. Kubernetes allows an empty apiGroups field but Openshift does not. Figured that adding it wouldn't hurt because everything will work on Kubernetes just the same even after the change.

Reply:
Yup, basically validation fails on OpenShift because it uses a bit tighter model